alien-root-legacy.sh

```diff
@@ -40,6 +43,11 @@ for RPKG in $BUILD_REQUIRES; do
   rm -f $INSTALLROOT/etc/modulefiles/{$RPKG,$RPKG.unrelocated} || true
 done
 
+if [[ $ARCHITECTURE == osx* ]]; then
+  ln -nfs $INSTALLROOT/lib/libAliEnROOTLegacy.dylib $INSTALLROOT/lib/libAliEnROOTLegacy.so
```

Comment:
The two symlinks here will not be relocated at install time, please use relative paths:

```bash
ln -nfs libBlahBlah.dylib $INSTALLROOT/lib/libBlahBlah.so
```